static/styles/settings.css

```diff
@@ -276,9 +276,25 @@ td .button {
         z-index: 1;
     }
 
+    #id_realm_authentication_methods {
+        width: 20%;
+
+        th,
+        td {
+            border-collapse: collapse;
+        }
```

Comment:
Why do we need since border is set to None? Also there is no `th` element in this table.